GettingStartedGuide.md

```diff
@@ -45,6 +45,7 @@ Once completed, one can progress to the [Use Security Features](UseSecurityFeatu
 - WiFi access point with access to the internet.
 
 ### 1.2 Software Requirements
+> **NOTE:** For convenience sake, there is an esp-idf v5.1.1 submoduled at the root of the directory, other versions on esp-idf may also be used but does not guarantee full compatibility.
 
 - ESP-IDF 4.4.3 or higher to configure, build, and flash the project. To setup for the ESP32-C3, follow Espressif's [Getting Started Guide for the ESP32-C3](https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/get-started/index.html).
```

Comment:
This is incorrect now if we're telling users to use 5.1.1

Reply:
Well we still want people to use other version of esp-idf but just are saying the full support will be only for 1. Might help us with ops burden.